generator/Makefile

```diff
@@ -27,16 +27,19 @@ FRAMEWORK_URL    := http://www.net-snmp.org/docs/mibs/SNMP-FRAMEWORK-MIB.txt
 IANA_CHARSET_URL := https://www.iana.org/assignments/ianacharset-mib/ianacharset-mib
 IANA_IFTYPE_URL  := https://www.iana.org/assignments/ianaiftype-mib/ianaiftype-mib
 IANA_PRINTER_URL := https://www.iana.org/assignments/ianaprinter-mib/ianaprinter-mib
+IPV6_TC_URL      := http://www.net-snmp.org/docs/mibs/IPV6-TC.txt
+ISDN_URL         := https://raw.githubusercontent.com/librenms/librenms/master/mibs/ISDN-MIB
```

Comment:
Where's the official source for this? librenms copies from elsewhere.

Reply:
I can not find it.
It may be possible to extract from RCF documents.

Reply:
I think we're already pulling this in from elsewhere.

Reply:
Thank you! I missed it.